### PR TITLE
1.0 Documentation: Fetching Data

### PR DESCRIPTION
**Pull request by AnthonyMDev**
This PR includes all the documentation updates for articles related to fetching data.

Structure of the side bar is still a work in progress.

The following pages are complete as of this PR:
- Fetching Data
- Queries
- Mutations
- Subscriptions
- Fragments
- Operation Arguments
- Error Handling
- Type Conditions
- Automatic Persisted Queries

### CODE REVIEW

**Review by bignimbus — COMMENTED** [review]
This is amazing @AnthonyMDev - so much quality content!  I learned a lot reading them and left some suggestions, most of which are pretty low-level.  I hope that helps!  I know the community will benefit a ton from these very informative docs

File: docs/source/fetching/apqs [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+title: Automatic Persisted Queries
```
[the remark this answers is not in the document]
Comment (AnthonyMDev) [reply docs/source/fetching/apqs]: Thanks for noticing this! Fixed!

File: docs/source/fetching/fragments.mdx [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+    }
+  }
+}
+```
+
+You can add additional data to the `HeroDetails` fragment and display that data in the `HeroDetailsView` without affecting the parent view at all. Additionally, the `HeroDetailsView` can be used with other GraphQL operations that include the `HeroDetails` fragment.
+
+## Inline fragments
+
+An [inline fragment](http://graphql.org/learn/queries/#inline-fragments) is a fragment that has no name. It is declared "inline" within another definition and cannot be reused. Inline fragments can be included in an operation definition, a named fragment definition, or nested inside another inline fragment. They are typically used to conditionally include fields in an operation using a [type conditions](./type-conditions) or the `@skip`/`@include` directives.
```
[the remark this answers is not in the document]
Comment (AnthonyMDev) [reply docs/source/fetching/fragments.mdx:133]: Not yet, I'm expecting to add one and I'll link it when I do. (May not get done this week)

File: docs/source/fetching/operation-arguments.mdx [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+
+  init(episode: GraphQLNullable<Episode>) {
+    self.episode = episode
+  }
+}
+```
+
+The `HeroNameQuery` can be initialized with a `GraphQLNullable` value:
+
+<CodeColumns cols={3}>
```
[the remark this answers is not in the document]
Comment (AnthonyMDev) [reply docs/source/fetching/operation-arguments.mdx:66]: 😉

**Review by AnthonyMDev — COMMENTED** [review]
Thanks for the review @bignimbus!

**Review by jpvajda — COMMENTED** [review]
I had some minor feedback.

File: docs/source/code-generation.mdx [hunk 4]
```diff
@@ -1,5 +1,31 @@
...
+To generate classes, Apollo iOS requires your server's schema, along with all of the `.graphql` files that contain your defined operations. It uses these to generate code you can use to execute queries and access typed results.
+
+All `.graphql` files in your project (or the subset you specify as input to the `Codegen CLI` if you customize your code generation configuration) will be combined and treated as one big GraphQL document.
+
+This means that fragments defined in one .graphql file are available across all .graphql files. However, it also means that operation and fragment names must be unique (and validation errors will occur if they aren't).
+
+
+## Learn what you need to generate code
+
+Now that you've got the SDK set up, there are two more pieces you need to actually generate code.
```
[the remark this answers is not in the document]
Comment (AnthonyMDev) [reply docs/source/code-generation.mdx:14]: This page has not been updated yet. I'm going to work on it today. Everything in here is old docs about the OLD codegen... So don't use this in the Blog post lol.

Pages that have # UNDER CONSTRUCTION at the top are not ready yet for review.

**Pull request merged**